sql/database.go

```diff
@@ -415,6 +720,13 @@ func (db *sqliteDatabase) WithTxImmediate(
 // Note that Exec will block until database is closed or statement has finished.
 // If application needs to control statement execution lifetime use one of the transaction.
 func (db *sqliteDatabase) Exec(query string, encoder Encoder, decoder Decoder) (int, error) {
+	if err := db.runInterceptors(query); err != nil {
+		return 0, err
```

Comment:
would be nice to have error instrumentation here

Reply:
Fixed